bin/server/cmd/root.go

```diff
@@ -91,6 +93,9 @@ var (
 	readonly bool
 	demo     bool
 	debug    bool
+	// pgURL must follow PostgreSQL connection URIs pattern.
+	// https://www.postgresql.org/docs/current/libpq-connect.html#LIBPQ-CONNSTRING
+	pgURL string
```

Comment:
I prefer that we take host/port/user/password information separately for a better user experience (less think). And not all parameters in pg urls are supported such as SSL that requires registration.